pkgs/by-name/bu/bustle/package.nix

```diff
@@ -32,6 +36,14 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-r29Z+6P+yuCpOBUE3vkESd15lcGXs5+ZTBiQ9nW6DJ4=";
   };
 
+  # Use gettext from nixpkgs instead of the vendored one, which does not build on Darwin with clang 16.
+  env = {
+    GETTEXT_BIN_DIR = lib.getBin buildPackages.gettext;
+    GETTEXT_INCLUDE_DIR = lib.getDev gettext;
+    GETTEXT_LIB_DIR = lib.getLib gettext;
```

Comment:
I was unable to find docs for `cargo:{lib,include`` but should not these include `/{bin,include/lib}` suffixes?

Reply:
It’s getting late, so I’ll have to look at it in the morning. The builds succeeded, so either those aren’t needed, or something else is happening.

Reply:
I updated the PR to make it conditional on Darwin and specify the correct paths. Since the `gettext-system` feature is enabled in upstream’s `Cargo.lock`, I revised the comment and dropped setting the `GETTEXT_SYSTEM` variable.